tests/Integration/ClientMiddlewareTest.php

```diff
@@ -79,7 +79,7 @@ public function testAuthenticationRetryFails() : void
         ]);
 
         $this->expectException(RequestFailed::class);
-        $this->expectExceptionMessage("User 'ghost' is not found");
+        $this->expectExceptionMessageMatches("(User 'ghost' is not found|User not found or supplied credentials are invalid)");
```

Comment:
Although `//` delimiters are not strictly necessary for this simple regex, let's add them for consistency with other similar calls (and to avoid potential problems when rewriting it into something more complex; for example, `(a|b)!` will not compile).

```suggestion
        $this->expectExceptionMessageMatches("/(User 'ghost' is not found|User not found or supplied credentials are invalid)/");
```

Reply:
Done.